core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/wordCloud/WordCloudOpDesc.java

```diff
@@ -25,18 +26,26 @@
  */
 
 public class WordCloudOpDesc extends VisualizationOperator {
-    @JsonProperty(value = "text column", required = true)
+    @JsonProperty(required = true)
+    @JsonSchemaTitle("Text column")
     @AutofillAttributeName
     public String textColumn;
 
+    @JsonProperty(required = false, defaultValue = "100")
+    @JsonSchemaTitle("Number of most frequent words")
+    public Integer topN;
+
     @Override
     public String chartType() {
         return VisualizationConstants.WORD_CLOUD;
     }
 
     @Override
     public OpExecConfig operatorExecutor() {
-        return new WordCloudOpExecConfig(this.operatorIdentifier(), Constants.defaultNumWorkers(), textColumn);
+        if (topN == null) {
+            topN = 100;
+        }
```

Comment:
the input should be positive number only. frontend ensures that it is not null, but we did not check for negative or 0 case. 

negative int will give exception.

Reply:
added check in json schema